CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Welcome to the Fedimint project! We're thrilled that you're interested in contributing to our open-source community. Below are some resources to help you get started:
+
+- **[Contributing Guidelines](https://github.com/fedimint/fedimint/blob/master/docs/contributing.md)**: Read our guidelines to understand how you can contribute to Fedimint.
```

Comment:
Let's not have this file link to itself

Reply:
The document mentioned is in a different folder from the `CONTRIBUTING.md` file in the main repository.

Reply:
just move the contributing.md out of the docs folder into this root, link to it from the readme, and update the instructions to new contributors to include what I put in the discord under the new-contributors channel

Reply:
Afaik having it in the root is better anyway since GitHub can detect it and might have some current or future integrations with it.